src/dataMap/metaManager/utils.js

```diff
@@ -1,5 +1,5 @@
 import { hasOwnProperty, isObject, objectEach, inherit } from '../../helpers/object';
-import { getCellType } from '../../cellTypes';
+import { getCellType } from '../../cellTypes/cellTypes';
```

Comment:
The same as [here](https://github.com/handsontable/handsontable/pull/7403/files#diff-8ab41fe13597e1554b5d6b4c227b5f123ff2d6726a7f3688a8b8d1224fe1d4f3R20).

Reply:
I'll reply there